acquia_cms.info.yml

```diff
@@ -6,7 +6,7 @@ distribution:
   install:
     finish_url: '/admin/tour'
 install:
-  - acquia_cms_common
+  - acquia_cms_page
```

Comment:
We should keep both `acquia_cms_common` & `acquia_cms_page`

Reply:
Common shouldn't be needed since it is required by the other component modules.